src/test/groovy/org/prebid/server/functional/tests/privacy/GppFetchBidActivitiesSpec.groovy

```diff
@@ -689,6 +708,263 @@ class GppFetchBidActivitiesSpec extends PrivacyBaseSpec {
+
+        where:
+        gpcValue | accountLogic
+        true     | LogicalRestrictedRule.generateSolidRestriction(OR, [])
+        false    | LogicalRestrictedRule.generateSolidRestriction(OR, [new ValueRestrictedRule(true, GPC, CONSENT)])
+        true     | LogicalRestrictedRule.generateSolidRestriction(OR, [new ValueRestrictedRule(false, GPC, CONSENT)])
+        true     | LogicalRestrictedRule.generateSolidRestriction(AND, [new ValueRestrictedRule(true, GPC, CONSENT),
+                                                                        new ValueRestrictedRule(true, SHARING_NOTICE, CONSENT)])
+    }
+
+    def "PBS auction call when privacy regulation match custom requirement should ignore call to bidder"() {
+        given: "Default basic generic BidRequest"
+        def gppConsent = new UspNatV1Consent.Builder().setGpc(gpcValue).build()
+        def accountId = PBSUtils.randomNumber as String
+        def generalBidRequest = BidRequest.defaultBidRequest.tap {
+            regs.gppSid = [USP_NAT_V1.intValue]
+            regs.gpp = gppConsent
+            setAccountId(accountId)
+        }
+
+        and: "Activities set for fetch bid with allowing privacy regulation"
+        def rule = new ActivityRule().tap {
+            it.privacyRegulation = [IAB_US_CUSTOM_LOGIC]
+        }
+
+        def activities = AllowActivities.getDefaultAllowActivities(FETCH_BIDS, Activity.getDefaultActivity([rule]))
+
+        and: "Account gpp configuration with sid skip"
+        def accountGppConfig = new AccountGppConfig().tap {
+            it.code = IAB_US_CUSTOM_LOGIC
+            it.config = new SidsConfig().tap { it.skipSids = [] }
+            it.enabled = true
+            it.moduleConfig = ModuleConfig.getDefaultModuleConfig(new ActivityConfig([FETCH_BIDS], accountLogic), [USP_NAT_V1], false)
+        }
+
+        and: "Existed account with privacy regulation setup"
+        def account = getAccountWithAllowActivitiesAndPrivacyModule(accountId, activities, [accountGppConfig])
+        accountDao.save(account)
+
+        when: "PBS processes auction requests"
+        activityPbsService.sendAuctionRequest(generalBidRequest)
+
+        then: "Generic bidder request should be ignored"
+        assert bidder.getBidderRequests(generalBidRequest.id).size() == 0
+
+        where:
+        gpcValue | accountLogic
+        true     | LogicalRestrictedRule.generateSolidRestriction(OR, [new ValueRestrictedRule(true, GPC, CONSENT)])
+        false    | LogicalRestrictedRule.generateSolidRestriction(OR, [new ValueRestrictedRule(false, GPC, CONSENT)])
+        true     | LogicalRestrictedRule.generateSolidRestriction(OR, [new ValueRestrictedRule(true, GPC, CONSENT),
+                                                                       new ValueRestrictedRule(true, SHARING_NOTICE, CONSENT)])
+    }
+
+    def "PBS auction call when custom privacy regulation have invalid setup should call to bidder with warning and add alert metric"() {
```

Comment:
Agree but we have this in the [description](https://github.com/prebid/prebid-server/issues/2686#Prebid%20Server-Specific%20Requirements):
```
Errors in the account-level JSON should cause the system to either fail every request, or, preferably, fall back to the host-level default. We don't want to skip broken sections because that could wind up creating a legal problem. A failure should cause the system should generate an alert.general metric with N% logging and emit a warning in the ORTB response.
```